migrations/20200206005201-create-patient.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('patients', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      gender: {
+        type: Sequelize.STRING
+      },
+      age: {
+        type: Sequelize.INTEGER
+      },
+      runNumber: {
+        type: Sequelize.BIGINT
+      },
+      barcodeValue: {
+        type: Sequelize.BIGINT
+      },
+      collectionPointId: {
```

Comment:
let's just call this `collectionPoint` instead
```suggestion
      collectionPoint: {
```